oci/cas/cas.go

```diff
@@ -42,6 +42,11 @@ var (
 	// ErrInvalid is returned when an image was detected as being invalid.
 	ErrInvalid = fmt.Errorf("invalid image detected")
 
+	// ErrUnknownType is returned when a mediatype was encountered that is not
+	// known how to parse. According to the spec, any caller of a function that
```

Comment:
the phrasing here is strange, what about: "ErrUnkownType is returned when an unparsable mediatype is encountered "